sandbox/graph-config-tooltips.js

```diff
@@ -1,125 +1,127 @@
 /*eslint-disable*/
 export const tooltips = {
```

Comment:
Please discard these changes; this file is only meant to be updated upon releasing a new version.

Reply:
Same for `sandbox/rd3g.sandbox.bundle.js`

Reply:
Ah I see. I ran some npm script that did it.